app/main.py

```diff
@@ -1,29 +1,29 @@
 class Person:
     people = {}
 
-    def __init__(self, name, age, wife=None, husband=None):
+    def __init__(self, name: str, age: int):
         self.name = name
         self.age = age
-        if wife is not None:
-            self.wife = wife
-        if husband is not None:
-            self.husband = husband
-        self.__class__.people[self.name] = self
+        self.people[self.name] = self
+
+    @classmethod
+    def is_wife(cls, person):
+        if "wife" in person and person["wife"] is not None:
+            if person["wife"] in cls.people:
+                cls.people[person["name"]].wife = cls.people[person["wife"]]
+                cls.people[person["wife"]].husband = cls.people[person["name"]]
+
+    @classmethod
+    def is_husband(cls, pers):
+        if "husband" in pers and pers["husband"] is not None:
+            if pers["husband"] in cls.people:
+                cls.people[pers["name"]].husband = cls.people[pers["husband"]]
+                cls.people[pers["husband"]].wife = cls.people[pers["name"]]
 
 
 def create_person_list(people: list) -> list:
     for person in people:
-        Person(
-            person["name"],
-            person["age"],
-            wife=person["wife"] if "wife" in person else None,
-            husband=person["husband"] if "husband" in person else None
-        )
-    for person in people:
-        if "wife" in person and person["wife"] is not None:
-            Person.people[person["name"]].wife = \
-                Person.people[person["wife"]]
-        if "husband" in person and person["husband"] is not None:
-            Person.people[person["name"]].husband = \
-                Person.people[person["husband"]]
+        Person(person["name"], person["age"])
```

Comment:
You can use one "for" for add people on Person.people and another one for add wife/husband, so you can delete $12, $14, $19, $21 but i dont think this is better than your version